easybuild/toolchains/iomkl.py

```diff
@@ -31,14 +31,15 @@
 """
 
 from easybuild.toolchains.iompi import Iompi
+from easybuild.toolchains.iimkl import Iimkl
 from easybuild.toolchains.fft.intelfftw import IntelFFTW
 from easybuild.toolchains.linalg.intelmkl import IntelMKL
 
 
-class Iomkl(Iompi, IntelMKL, IntelFFTW):
+class Iomkl(Iompi, Iimkl, IntelMKL, IntelFFTW):
```

Comment:
Since your inheriting from `Iimkl` you don't need to also inherit from `IntelMKL, IntelFFTW` (they're already there)